src/App.css

```diff
@@ -88,20 +123,287 @@ h2 {
 }
 
 .book-item {
-  margin-top: 1vw;
+  width: 77vw;
+  height: 10.625vw;
+  margin: 4vw 0 0 0;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
```

Comment:
- [ ] In this section, kindly remove the line labelled 1 on the RHS of the edit button
- [ ] It would be nice if you could make the progress bar labelled 2, the same size as the one in the zeplin design for a better UI
- [ ] There is a black outline on the border of your `Update Progress` button labelled 3, kindly remove it to fit the design, you can do this by giving the button a style of `border: none;`

### Your design
<img width="1109" alt="Screenshot 2023-07-08 at 21 04 49" src="https://github.com/JuanDBta/bookstore-project/assets/64774936/09b8c1ce-e3c2-422c-9b68-670287ae55dc">

### Zeplin design
![Screenshot 2023-07-08 at 21 04 49 (2)](https://github.com/JuanDBta/bookstore-project/assets/64774936/28e2440b-3422-47cf-b197-ac78324ddc87)